src/Simulation/Simulators.Tests/TestProjects/QSharpExe/QSharpExe.csproj

```diff
@@ -11,7 +11,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\..\CSharpGeneration\Microsoft.Quantum.CSharpGeneration.fsproj" PrivateAssets="All" IsQscReference="true" />
+    <PackageReference Include="Microsoft.Quantum.CSharpGeneration" Version="0.17.2106147878-beta" IsQscReference="true" />
```

Comment:
If we remove `<CSharpGeneration>false</CSharpGeneration>`, can we remove this `PackageReference`?

Reply:
Because this needs to include the locally compiled `Microsoft.Quantum.Runtime.Core.csproj` I don't have a choice and need to use `<CSharpGeneration>false</CSharpGeneration>`, which then requires that I use a package reference to CSharpGeneration. I'm looking into how to centralize these version numbers so I don't start introducing the same problem we worked to remove with the QDK version numbers.

Reply:
This is now resolved and uses the right packages from the right places.